hqmedia.upload_controller.js

```diff
@@ -182,19 +178,11 @@ function BaseHQMediaUploadController (uploader_name, marker, options) {
     // Uploader flow
     self.init = function () {
         /*
-            Initialize the uploader.
-
-            Here we use YUI for the uploader. Flash is required.
-            We tried non Flash at some point and gave up after a myriad of issues.
+            Initialize the YUI uploader.
+            Use HTML5 version; flash version caused problems.
```

Comment:
Would it make sense to elaborate on what kind of problems it caused? We may find that the HTML5 version causes problems too, and it could be nice to be able to compare the problems to know if they are the same or which set is worse.